### PR TITLE
read excitation during restart and at init

### PR DESCRIPTION
**Pull request by RevathiJambunathan**
This PR is a potential fix to the restart Issue #40

### CODE REVIEW

**Review by ajnonaka — APPROVED** [review]
It passes the broken pure-EM test problems on branch restart_test

File: Examples/Tests/circuits/inputs_roger_pec_restarttest [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 1
+warpx.use_filter = 0
+warpx.cfl = 0.8
+boundary.field_lo = periodic pec pec   # unrealistic boundary conditions in x and z just for restart test
+boundary.field_hi = periodic pml pml
+boundary.field_lo = periodic periodic periodic
+boundary.field_hi = periodic periodic periodic
```
Comment (ajnonaka) [inline Examples/Tests/circuits/inputs_roger_pec_restarttest]:
```suggestion
```
